project/images/official/editable/464.svg

```diff
@@ -40,136 +40,120 @@
+    <path
+       fill="#007d45"
+       fill-rule="nonzero"
+       id="svg_16"
+       d="m 112.92413,5.18541 c -2.47189,0 -4.47613,1.99118 -4.47613,4.44695 v 85.36518 c 0,2.45068 2.00424,4.44185 4.47613,4.44185 h 110.06881 c 2.4719,0 4.471,-1.99118 4.471,-4.44185 V 9.63236 c 0,-2.45577 -1.9991,-4.44695 -4.471,-4.44695 H 112.92413" />
+    <path
+       fill="#007d45"
+       fill-rule="nonzero"
+       id="svg_15"
+       d="m 93.24145,99.43939 c 2.4719,0 4.47615,-1.99118 4.47615,-4.44185 V 9.63236 c 0,-2.45068 -2.00424,-4.44695 -4.47615,-4.44695 H 10.01919 c -2.47189,0 -4.47613,1.99627 -4.47613,4.44695 v 85.36518 c 0,2.45068 2.00424,4.44185 4.47613,4.44185 h 83.22226" />
+    <path
+       fill="#ffffff"
+       fill-rule="nonzero"
+       id="svg_14"
+       d="M 33.25811,90.5557 15.34842,89.28952 c -0.29806,-0.02553 -0.56529,-0.1889 -0.70919,-0.44419 -0.14903,-0.25528 -0.16445,-0.56671 -0.03084,-0.83221 l 7.84738,-16.04681 c 0.16445,-0.33187 0.5139,-0.53098 0.88392,-0.49525 0.37001,0.03064 0.6835,0.28591 0.78114,0.63821 l 2.02994,7.05079 16.71229,-10.35921 3.35582,5.77439 -17.38037,9.20535 5.12881,5.27406 c 0.25695,0.26039 0.32376,0.65862 0.16445,0.99048 -0.15418,0.33696 -0.50364,0.53609 -0.87365,0.51056" />
+    <path
+       fill="#ffffff"
+       fill-rule="nonzero"
+       id="svg_13"
+       d="m 199.72319,90.56591 17.90969,-1.26618 c 0.29808,-0.02553 0.56017,-0.1889 0.71433,-0.44418 0.1439,-0.25529 0.15932,-0.56673 0.0257,-0.83221 l -7.84737,-16.04682 c -0.16445,-0.33185 -0.5139,-0.53098 -0.88392,-0.49523 -0.37001,0.03064 -0.67837,0.2808 -0.78114,0.6382 l -2.02994,7.05079 -16.71743,-10.35921 -3.35583,5.7744 17.39065,9.20535 -5.13394,5.26895 c -0.25695,0.26548 -0.32376,0.66372 -0.16959,0.99558 0.15932,0.33698 0.50877,0.5412 0.87878,0.51056" />
+  </g>
+  <text
+     xml:space="default"
+     dominant-baseline="middle"
+     text-anchor="middle"
+     font-family="sans-serif"
+     font-size="14.5989px"
+     id="svg_24"
+     y="30.836546"
+     x="52.386219"
+     fill="#ffffff"
+     style="stroke-width:0.973261">param(text1)</text>
+  <text
+     xml:space="default"
+     dominant-baseline="middle"
+     text-anchor="middle"
+     font-family="sans-serif"
+     font-size="14.5989px"
+     id="svg_25"
+     y="50.301762"
+     x="52.386219"
+     fill="#ffffff"
+     style="stroke-width:0.973261">param(text2)</text>
+  <text
+     xml:space="default"
+     dominant-baseline="middle"
+     text-anchor="middle"
+     font-family="sans-serif"
+     font-size="14.5989px"
+     id="svg_26"
+     y="30.836546"
+     x="163.33795"
+     fill="#ffffff"
+     style="stroke-width:0.973261">param(text3)</text>
+  <text
+     xml:space="default"
+     dominant-baseline="middle"
+     text-anchor="middle"
+     font-family="sans-serif"
+     font-size="14.5989px"
+     id="svg_27"
+     y="50.301762"
+     x="163.33795"
+     fill="#ffffff"
+     style="stroke-width:0.973261">param(text4)</text>
```

Comment:
we don't have any `inscription_4` in the data model. Should we add it?

Reply:
yeah either add a new inscription_4 field or set `no_dynamic_inscription` to 4 for this one.